### ISSUE TITLE
OpenAPI 3.0 JSONSchema has been released

### ISSUE THREAD

**Issue by intelliot** [kogosoftwarellc/open-api#451]
The [README](https://github.com/kogosoftwarellc/open-api/blob/fbdae0b/packages/express-openapi/README.md) currently states:
> OpenAPI 3.0 (beta until the official JSONSchema is released https://github.com/OAI/OpenAPI-Specification/issues/1301)

The schema is now available:
https://github.com/OAI/OpenAPI-Specification/tree/master/schemas/v3.0

Linked from:
https://github.com/OAI/OpenAPI-Specification/issues/1032#issuecomment-492410088

**Comment by jsdevel** [comment]
good catch!  open to anyone opening a pr to update the schema and the documentation

**Comment by MikeyUchiha** [comment]
@jsdevel  What's the status on this update? Thanks

**Comment by jsdevel** [comment]
> @jsdevel What's the status on this update? Thanks

https://github.com/kogosoftwarellc/open-api/issues/451#issuecomment-492868616

**Comment by jameskhedley** [comment]
@jsdevel can you give me some more details on how to "update the schema"? What schema are you referring to? The link above takes you to `schema.json` but there's nothing like that in this repo.

**Comment by jsdevel** [comment]
https://github.com/kogosoftwarellc/open-api/tree/master/packages/openapi-schema-validator

**Comment by jsdevel** [comment]
it's likely not as simple as just replacing the json schema file though... we've been basing _everything_ on the schema in that schema validator project i linked to.

**Comment by jsdevel** [comment]
openapi-framework used that schema validator package: https://github.com/kogosoftwarellc/open-api/blob/master/packages/openapi-framework/index.ts

and then every other package assumes the incoming config etc. is all based on the existing v3 schema.

i'm fine proceeding though with the official schema, but it'll likely be a major version bump, and will halt development on the existing schema.  not the biggest deal but worth mentioning... we _should_ use the official schema.

**Comment by philsturgeon** [comment]
You should be able to bump the schema without a major, we did it for https://github.com/stoplightio/spectral and nothing exploded.

**Comment by nasht00** [comment]
Any news on this?

Also, to clarify, what does "beta" mean? If I were to use a 3.0 schema today using express-openapi, what would happen?
